acmwebsite/templates/meeting_list.xhtml

```diff
@@ -3,8 +3,8 @@
     <div class="acm-meeting-block">
       <span class="acm-meeting-title" py:content="meeting.title" />
       <span py:if="meeting.description"> – ${meeting.description}</span><br />
-      <b>Date:</b> <span class="acm-meeting-date" py:content="h.ftime(meeting.date, show_day=True)" /><br />
-      <b>Location:</b> <span class="acm-meeting-location" py:content="meeting.location" />
+      <strong>Date:</strong> <span class="acm-meeting-date" py:content="h.ftime(meeting.date, show_day=True)" /><br />
+      <strong>Location:</strong> <span class="acm-meeting-location" py:content="meeting.location" />
```

Comment:
Why `strong` rather than `b`?

Reply:
The `strong` tag is the Semantic HTML way of specifying strong emphasis (rendered as bold). `<strong>` is better for accesibility (think screen readers).

Reply:
This is not for strong emphasis, it's for display only... hence, should not provide any meaningful for screen readers. `strong` would be incorrect here.